src/shared/lib/GenericAssayUtils/GenericAssayCommonUtils.ts

```diff
@@ -7,8 +7,14 @@ import {
     GenericAssayFilter,
 } from 'cbioportal-ts-api-client';
 import { MolecularProfile } from 'cbioportal-ts-api-client';
-import _, { Dictionary } from 'lodash';
+import _, { capitalize, Dictionary } from 'lodash';
```

Comment:
seems strange that we are importing capitalize rather than just using ```_.capitalize```
Also, how come we use lodash Dictionary instead of native Map?  I mean maybe there is a good reason but just wondering